session/tidb.go

```diff
@@ -251,6 +251,13 @@ func runStmt(ctx context.Context, sctx sessionctx.Context, s sqlexec.Statement)
 		defer span1.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
+	sctx.SetValue(sessionctx.QueryString, s.OriginText())
```

Comment:
we don't need set `LastExecuteDDL` any more?

Reply:
Do it in line255-259.